eth-contracts/contracts/DelegateManager.sol

```diff
@@ -338,18 +338,19 @@ contract DelegateManager is InitializableV2 {
         (
             uint totalBalanceInStaking,
             uint totalBalanceInSPFactory,
-            uint totalBalanceOutsideStaking
+            uint totalBalanceOutsideStaking,
+            uint spLockedStake,
+            uint totalRewards
         ) = _validateClaimRewards(spFactory);
 
         // No-op if balance is already equivalent
         // This case can occur if no rewards due to bound violation or all stake is locked
-        if (totalBalanceInStaking == totalBalanceOutsideStaking) {
+        if (totalRewards == 0) {
```

Comment:
is this change bc previously `totalBalanceOutsideStaking = totalBalanceInSPFactory - spLockedUpStake + totalDelegatedStake` which could equal 0 but now that it no longer subs `spLockedUpStake` it will always be > 0 so you instead just compare directly to 0?

Reply:
no, its bc this is the same calculation, totalRewards is equal to the diff but wasn't returned prior